jekyll/_cci2/roles-and-permissions-overview.adoc

```diff
@@ -426,6 +456,41 @@ The table below shows the permissions associated with each CircleCI project role
 ^|
 ^|
 
+| *Releases*
+|
+|
+|
+
+^| Restore component version
+^| icon:check-circle[]
+^| icon:check-circle[]
+^|
+
+^| Restart Component
+^| icon:check-circle[]
+^| icon:check-circle[]
+^|
+
+^| Scale component
```

Comment:
Question: Should this and the "restart" action refer to component _versions_ rather than just components?

Reply:
those are going to be moved to the component level https://circleci.atlassian.net/browse/REL-2402